policy/modules/system/systemd.if

```diff
@@ -161,6 +162,8 @@ template(`systemd_role_template',`
 
 	can_exec($3, systemd_analyze_exec_t)
 
+	domtrans_pattern($1_systemd_t, systemd_tmpfiles_exec_t, systemd_tmpfiles_t)
```

Comment:
The systemd user sessions definitely shouldn't be starting a system service.

Reply:
I'm open to suggestions on this one.  I started down the path of setting up a user tmpfiles concept (i.e. setting up $1_systemd_tmpfiles_t), but it quickly got out of hand on permissions.  I can try this approach again unless there is a better alternative?

Reply:
I agree transitioning from the systemd user session to the tmpfiles domain directly is a pretty significant elevation of privileges.

What kinds of things does this operation need to do? I'm not familiar with the innerworkings of cockpit and how this is related.

Reply:
Ok, after reviewing this, it does seem that $1_systemd_tmpfiles_t is working ok and not an unreasonable amount of policy.  Maybe I was looking at this too soon during the development of the overall cockpit poilcy and was seeing something else that was confusing me.